docs/tutorials/00-local-setup.md

```diff
@@ -0,0 +1,163 @@
+# Deploy and Use the Landscaper locally
+
+This tutorial describes how a landscaper can be deployed locally.
+The only prerequisite is a kubernetes cluster 
+(find [below](#kubernetes-clusters) a list of possible local kubernetes installers but basically every k8s should work).
+
+### Install a container registry
+The Landscaper depends on an oci compliant registry to fetch Blueprints, Component Descriptors and other artifacts.
+[Harbor](https://github.com/goharbor/harbor-helm) can be used as such a registry that can also run beside the Landscaper in a kubernetes cluster.
+
+Configure the harbor with a `values.yaml` file.
+See additional configuration options [here](https://github.com/goharbor/harbor-helm).
+```
+# values.yaml
+registry:
+  credentials:
+    username: "harbor_registry_user"
+    password: "harbor_registry_password"
+    # If you update the username or password of registry, make sure use cli tool htpasswd to generate the bcrypt hash
+    # e.g. "htpasswd -nbBC10 $username $password"
+    htpasswd: "harbor_registry_user:$2y$10$9L4Tc0DJbFFMB6RdSCunrOpTHdwhid4ktBJmLD00bYgqkkGOvll3m"
+chartmuseum:
+  enabled: false
+clair:
+  enabled: false
+trivy:
+  enabled: false
+notary:
+  enabled: false
+```
+
+```
+helm repo add harbor https://helm.goharbor.io
+helm upgrade --install -n ls-system registry harbor/harbor -f values.yaml
+```
+
+Helm installs the registry in the `ls-system` namespace.
+The registry can be used by the landscaper by referencing artifacts via it's internal service `harbor.ls-system:80`
```

Comment:
At least with the helm installation above, there is no service with the name `harbor.<namespace>`.  I believe that should be `registry-harbor-registry.<namespace>:5000`.